csunplugged/tests/topics/loaders/test_lessons_loader.py

```diff
@@ -0,0 +1,457 @@
+import os.path
+from unittest.mock import Mock
+
+from tests.BaseTestWithDB import BaseTestWithDB
+from tests.topics.TopicsTestDataGenerator import TopicsTestDataGenerator
+
+from topics.models import Lesson
+from topics.management.commands._LessonsLoader import LessonsLoader
+
+from utils.errors.MissingRequiredFieldError import MissingRequiredFieldError
+
+
+class LessonsLoaderTest(BaseTestWithDB):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.load_log = Mock()
+        self.test_data = TopicsTestDataGenerator()
+        self.loader_name = "lessons"
+
+    def test_basic_config(self):
```

Comment:
I kind of expected this would actually check the configuration was applied. Maybe rename this test.